source/_includes/puppet-collections/_puppet_collections_intro.md

```diff
@@ -2,6 +2,31 @@
 > -   For Puppet 3.8 open source packages, see its [repository documentation](/puppet/3.8/puppet_repositories.html).
 > -   For Puppet Enterprise installation tarballs, see its [installation documentation](/pe/latest/install_basic.html).
 
-Puppet maintains official package repositories for several operating systems and distributions. To make the repositories more predictable, we version them as "Puppet Collections" --- each collection has all of the software you need to run a functional Puppet deployment, in versions that are known to work well with each other. Each collection is opt-in, and if you use `ensure => latest` to manage your Puppet packages, you’ll get the latest compatible versions in the collection you’re using. Whenever we make significant breaking changes that introduce incompatibilities between versions of our software, we make a new collection so that you can perform safe major upgrades at your convenience.
+Puppet maintains official package repositories for several operating systems and distributions. To make the repositories more predictable, we version them as "Puppet Collections" --- each collection has all of the software you need to run a functional Puppet deployment, in versions that are known to work well with each other. Each collection is opt-in, and you must choose a collection (and on some operating systems, install a package on Puppet-managed systems) to install software and receive updates from that collection.
```

Comment:
In the last sentence of this paragraph, 'collection' is used three times. Maybe it can be made more succinct?